server/src/main/java/org/apache/druid/server/QueryResource.java

```diff
@@ -163,7 +164,9 @@ public Response cancelQuery(@PathParam("id") String queryId, @Context final Http
   public Response doPost(
       final InputStream in,
       @QueryParam("pretty") final String pretty,
-      @Context final HttpServletRequest req // used to get request content-type,Accept header, remote address and auth-related headers
+
+      // used to get request content-type,Accept header, remote address and auth-related headers
```

Comment:
nit: `.. request content-type,Accept header, ..` -> `.. request content-type, Accept header, ..` 

(i know it was already like that, so imo only fix if you have something else to change).

Reply:
I'll update this if there ends up being something else to change, for now I'll leave it as it was before so CI doesn't get interrupted.